babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  presets: ['kcd-scripts/babel'],
+  plugins: ['@babel/transform-runtime'],
```

Comment:
This plugin is already in the built-in config.

I'm guessing this config is needed to compile `node_modules`? If that's the case, then I believe this should be sufficient:

```javascript
module.exports = require('kcd-scripts/babel')
```